audb/core/conftest.py

```diff
@@ -1,8 +1,35 @@
+import os
+
 import pytest
 
 import audb
 
 
+@pytest.fixture(scope='package', autouse=True)
+def cache(tmpdir_factory):
+    r"""Provide a reuseable cache for docstring tests.
+
+    As we rely on emodb from the public repo,
+    it makes sense to cache it
+    across all docstring tests.
+
+    """
+    cache = str(tmpdir_factory.mktemp('cache'))
```

Comment:
I think here and in many other places we could remove the `str(...)`. I guess it's a leftover from using `tmp_path_factory`. Or is the idea that we can easily switch to the other path library someday? But then we should maybe also use it with `tmpdir`.

Reply:
No, unfortunately `tmpdir_factory.mktemp()` does return some kind of path object, which works with some other functions (e.g. as `host` argument in `audb.Repository`), but in most cases we get the following error:

```
ERROR audb/core/api.py::audb.core.api.available - TypeError: str expected, not LocalPath
```

which is also true for this particular case.

Reply:
The cleanest solution seems to me to implement our own `tmp` fixture, but the downside is that you need to repeat it inside each `conftest.py` file.

Reply:
I was now able to avoid `str()` around `tmpdir_factory.mktemp()` and use it only at places where we really want to have a string.

Reply:
Thanks.